systemtest/src/main/java/io/strimzi/systemtest/performance/gather/collectors/BaseMetricsCollector.java

```diff
@@ -30,6 +40,20 @@ protected BaseMetricsCollector(Builder builder) {
         super(builder);
     }
 
+    @Override
+    protected String collectMetrics(String metricsPodIp, String podName) throws InterruptedException, ExecutionException, IOException {
```

Comment:
Isn't this duplicated to the method inside the `MetricsCollector` itself? I think you can make it protected or public there and just use it here or?

Reply:
Yeah, this method is a copy but removing unnecessary logging for performance tests. I was thinking how to do it...i.e., make a difference between our STs and performance ones. Because in classic STs we want logs like this
```java
Metrics collection for Pod: {}/{}({}) from Pod: {}/{} finished with return code: {}
...
// or creating KafkaUser n...
```
but in performance tests I think this is not needed at least now in the `INFO` level. So I have just override method and this change for Collector, which is used only for Performance. I think we should come up with better solution here :D

Reply:
Okay, I have come up with a solution here :) check the last commit :)